editor/editor_properties.cpp

```diff
@@ -108,6 +108,11 @@ void EditorPropertyText::update_property() {
 
 void EditorPropertyText::set_string_name(bool p_enabled) {
 	string_name = p_enabled;
+	if (p_enabled) {
+		Label *prefix = memnew(Label("&"));
```

Comment:
Can this label have a tooltip? Not sure it will be explicit enough what the `&` is without it.
```suggestion
		Label *prefix = memnew(Label("&"));
		prefix->set_tooltip_text("StringName");
```

Reply:
![image](https://github.com/godotengine/godot/assets/2223172/3780d442-9faa-433b-b6ef-fd18593a5ea4)